pkg/deviceplugin/manager.go

```diff
@@ -40,12 +40,17 @@ var DefaultConfig = NicManagerConfig{
 	NeedNic:        true,
 }
 
+type nic struct {
+	name string
+	numa int64
```

Comment:
nit: Maybe `int16` or `int`? The maximum `NODES_SHIFT` value on x86 seems to be 10, i.e. 1K NUMA nodes.

Reply:
Initially I had it as an `int`, but the spec for a [`NUMANode`](https://github.com/kubernetes/kubelet/blob/v0.28.3/pkg/apis/deviceplugin/v1beta1/api.pb.go#L308) from the deviceplugin api uses `int64`, so I thought being consistent was probably a easier than dealing with the conversions. I'm not anticipating running this on a machine with over 4 billion numa nodes!